interface/orders/single_order_results.inc.php

```diff
@@ -419,19 +418,14 @@ function generate_order_report($orderid, $input_form = false, $genstyles = true,
         padding: 4px;
```

Comment:
From line 372 to 420, change font-size property on all styles from `10pt` to `0.8125rem`. Also change border-color from`black` to `var(--black)` and `#e9e9e9` to `var(--gray200)`.

Reply:
Also add a `@media print` for all classes that contain 10pt so if it is printed you use pt instead of rem (Example in order_manifest.php lines 130 to 175)